docs/tutorial/kymographs.rst

```diff
@@ -267,3 +267,25 @@ The function can also take a dictionary of extra arguments to customize the kymo
 These parameter values get forwarded to :func:`matplotlib.pyplot.imshow`.
 
 .. image:: ./figures/kymographs/kymo_correlated.png
+
```

Comment:
I'd introduce `kymo.plot_with_channels` first. And after that introduce `kymo.plot_with_force` as a convenience function for a specific user case.